### PR TITLE
CORS-2478: GCP: add periodic jobs for XPN

### PR DESCRIPTION
**Pull request by AnnaZivkovic**
Create a periodic so we can observe the stability

### CODE REVIEW

**Review by bradmwilliams — APPROVED** [review]

File: ci-operator/config/openshift/release/openshift-release-master__nightly-4.13.yaml [hunk 1]
```diff
@@ -638,6 +638,12 @@ tests:
       enable:
       - observers-resource-watch
     workflow: openshift-e2e-gcp-realtime
+- as: e2e-gcp-ovn-xpn
+  interval: 168h
```
Comment (patrickdillon) [inline ci-operator/config/openshift/release/openshift-release-master__nightly-4.13.yaml:642]: Ideally I would like if we could bump this interval to ideally `8h`. I see that all the other intervals are running much less frequently. We would like to start getting regular periodics for this feature which we are trying to release in 4.13, but we could dial it back once we get a better signal or release.

**Review by patrickdillon — COMMENTED** [review]
@bradmwilliams your approve didn't take can you take another look? thx!

**Comment by bradmwilliams** [comment]
> @bradmwilliams your approve didn't take can you take another look? thx!

Unfortunately, I do not have any rights to `/approve` under `ci-operator/config/openshift/release`.  Looks like the best bet would be to reach out to your friendly neighborhood TRT member.

**Pull request merged**